black.py

```diff
@@ -464,15 +464,17 @@ def reformat_one(
         manager = Manager()
         lock = manager.Lock()
     tasks = {
-        loop.run_in_executor(
-            executor,
-            format_file_in_place,
-            src,
-            line_length,
-            fast,
-            write_back,
-            mode,
-            lock,
+        asyncio.create_task(
```

Comment:
create_task is new in 3.7, so we can't use it yet in Black, since we support 3.6 (https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task)